db/seeds.rb

```diff
@@ -7,17 +7,17 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
 Destination.create(continent: "North America", description: "North American continent description.",
-                    lat: 41.8369, long: -87.6847, image_url: "glacier.jpg" )
+                    lat: 41.8369, long: -87.6847, image_url: "south-america.jpg" )
 Destination.create(continent: "South America", description: "South American continent description.",
-                    lat: -42.8369, long: -17.6847, image_url: "Chile.jpg" )
+                    lat: -42.8369, long: -17.6847, image_url: "south-america.jpg" )
```

Comment:
Line is too long. [83/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.